src/unity.c

```diff
@@ -573,10 +573,7 @@ void UnityAssertEqualIntArray(UNITY_INTERNAL_PTR expected,
 
     RETURN_IF_FAIL_OR_IGNORE;
 
-    if (num_elements == 0)
```

Comment:
I'll follow your lead on the style.
I think I can just revert the Cleanup commit.